static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -94,13 +92,25 @@ function GroupRelatedIssues({params}: Props) {
               <HeaderWrapper>
                 <Title>{t('Issues caused by the same root cause')}</Title>
                 {sameRootCauseIssues.length > 0 ? (
-                  <GroupList
-                    endpointPath={`/organizations/${orgSlug}/issues/`}
-                    orgSlug={orgSlug}
-                    queryParams={{query: `issue.id:[${sameRootCauseIssues}]`}}
-                    query=""
-                    source="related-issues-tab"
-                  />
+                  <div>
+                    <TextButtonWrapper>
+                      <div />
+                      <LinkButton
+                        href={`/issues/?query=issue.id:[${groupId},${sameRootCauseIssues}]`}
+                        size="xs"
+                        external
```

Comment:
this link is not external

Reply:
@scttcper I'm trying to make it open on a new tab; should I avoid opening on a new tab?

Reply:
yeah i'd avoid opening a new tab and let people command click if they want it. most of our open in discover etc all open in the current tab